src/tracing.ts

```diff
@@ -27,12 +27,12 @@ export function enableTracing(service: Service, tracingMode: TracingMode, handle
       lambda: true,
     };
   }
-  if (tracingMode === TracingMode.HYBRID || tracingMode === TracingMode.DD_TRACE) {
-    handlers.forEach(({ handler }) => {
-      handler.environment ??= {};
-      const environment = handler.environment as any;
-      environment[ddTraceEnabledEnvVar] = true;
-      environment[ddMergeXrayTracesEnvVar] = tracingMode === TracingMode.HYBRID;
-    });
-  }
+  handlers.forEach(({ handler }) => {
+    handler.environment ??= {};
+    const environment = handler.environment as any;
+    // if tracing is not enabled, merge x-ray cannot be enabled
+    if (environment[ddTraceEnabledEnvVar] === false || environment[ddTraceEnabledEnvVar] === "false") {
```

Comment:
If a customer had specifically enabled xray merging on a function-level, would they have to set `ddTraceEnabled: true` as well as `ddMergeXrayTraces: true`?

Reply:
They wouldn't need to specifically set ddTraceEnabled because it will default to true, however, if they have that set as false and had set ddMergeXrayTraces to true (or DD_MERGE_XRAY_TRACES: true at the function-level), DD_MERGE_XRAY_TRACES will get set to false here instead since you can't have DD_MERGE_XRAY_TRACES: true and DD_TRACE_ENABLED: false. 

And they wouldn't need to set ddMergeXrayTraces: true if they've set DD_MERGE_XRAY_TRACES: true at the function level.

Reply:
AH gotchya. Sounds good!